httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/GGSSchemeBase.java

```diff
@@ -138,7 +236,9 @@ protected byte[] generateGSSToken(
         final GSSManager manager = getManager();
         final GSSName serverName = manager.createName(serviceName + "@" + authServer, GSSName.NT_HOSTBASED_SERVICE);
```

Comment:
I must admit that `authServer` is a horrible variable name.

Reply:
fixed